src/status-updates.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable  no-restricted-syntax */
+
+import { ToDoList } from './arr-crud'; // eslint-disable-line
+import { toDoListArr } from './index.js'; // eslint-disable-line
+import { setStorage } from './index.js'; // eslint-disable-line
+
+function checkboxesUpdate(currArr) {
```

Comment:
Kindly suggest you use modern js syntax.
```suggestion
const checkboxesUpdate = (currentArr) => {}
```